pkg/apis/aws/v1alpha1/awsfederatedrole_types.go

```diff
@@ -0,0 +1,117 @@
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// AWSFederatedRoleState defines the various status an AWSFederatedRole CR can have
+type AWSFederatedRoleState string
+
+const (
+	// AWSFederatedRoleStateInProgress const for InProgress status state
+	AWSFederatedRoleStateInProgress AWSFederatedAccountAccessState = "InProgress"
+	// AWSFederatedRoleStateVerified const for Requested status state
+	AWSFederatedRoleStateVerified AWSFederatedRoleState = "Valid"
```

Comment:
Whats the difference between `AWSFederatedRoleStateVerified` and `AWSFederatedRoleValid` since they both have the value `valid` ?

Reply:
@jharrington22 fixing this its a typo